src/main/java/net/syscon/elite/api/model/OffenceHistoryDetail.java

```diff
@@ -64,4 +64,7 @@
 
     @ApiModelProperty(value = "Latest court date associated with the offence", example = "2018-02-10", position = 14)
     private LocalDate courtDate;
+
+    @ApiModelProperty(value = "Court case id", example = "100", position = 15)
+    private String caseId;
```

Comment:
Just checking this is meant to be a string as ids are usually longs.

Reply:
Dam. Not had enough coffee yet!

Reply:
Never enough coffee for this stuff :D